unittests/Swift/BuildSystemBindingsTests.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  BuildSystemBindingsTests.swift
+//  llbuildSwiftTests
+//
+//  Copyright © 2019 Apple Inc. All rights reserved.
+//
+
+import XCTest
+
+// The Swift package has llbuildSwift as module
+#if SWIFT_PACKAGE
+import llbuild
+@testable import llbuildSwift
+#else
+@testable import llbuild
```

Comment:
I would like to avoid use of @testable, it prevents testing in release mode.

Reply:
@testable is necessary here due to internal initializer of `BuildKey` and `CommandExtendedResult` with their C-layer counterparts. I guess you're right about the necessity of those wiring tests in general, so we could remove those and remove @testable with it as well.